README.md

```diff
@@ -168,6 +168,24 @@ By default, `snyk-monitor` uses an emptyDir for temporary storage. If you prefer
 The PVC's name defaults to `snyk-monitor-pvc`. If you prefer to override this, then use the following value:
 * `pvc.name`
 
+## Configuring excluded namespaces ##
+
+By default, `snyk-monitor` does not scan containers that are internal to Kubernetes, in the following namespaces:
+* `kube-node-lease`
+* `kube-public`
+* `kube-system`
+* `local-path-storage`
+
+If you prefer to override this, you can add your own list of namespaces to exclude by setting the `excludedNamespaces` to your own list.
+For example:
+```yaml
+excludedNamespaces:
+  - kube-node-lease
+  - kube-public
+  - local-path-storage
+  - some_namespace
```

Comment:
I think based on https://helm.sh/docs/intro/using_helm/#the-format-and-limitations-of---set we can show example usage for deploying with Helm, e.g. `--set excludedNamespaces={kube-node-lease,some_namespace}`